addons/mail/static/src/discuss/call/common/call_settings.xml

```diff
@@ -23,11 +23,11 @@
                         <span class="ms-2 text-truncate">Enable Push-to-talk</span>
                     </label>
                 </div>
-                <div t-if="store.settings.use_push_to_talk and !pttExtService.isEnabled" class="mb-3">
-                    <span class="small text-muted fst-italic" t-out="pttExtService.downloadText"/>
+                <div t-if="store.settings.use_push_to_talk and !ui.isSmall and !pttExtService.isEnabled" class="mb-3">
+                    <span class="o-discuss-pttDownloadText small text-muted fst-italic" t-out="pttExtService.downloadText"/>
                 </div>
                 <t t-if="store.settings.use_push_to_talk">
-                    <div class="mb-3 d-flex align-items-center flex-wrap">
+                    <div t-if="!ui.isSmall" class="mb-3 d-flex align-items-center flex-wrap">
```

Comment:
`ui.isSmall` is not the same as touch-only / `isMobileOS()`. I know we sometimes make this mistake, but some users just like keeping many tabs with different size, some of which are narrow enough to become `isSmall` while still being shown on a screen with mouse/keyboard.

`ui.isSmall` should be as much as possible being used for space constraint, not UX/UI decision around the input method. `isMobileOS()` is decent for decision on input, or `@media` if solution can be done fully in SCSS.